README.rst

```diff
@@ -32,7 +32,7 @@ The documentation is hosted at https://docs.gunicorn.org.
 Installation
 ------------
 
-Gunicorn requires **Python 3.x >= 3.5**.
+Gunicorn requires **Python 3.x >= 3.7**.
```

Comment:
Note: Python 3.7 isn't strictly required for formatted string literals; they were [introduced in Python 3.6](https://docs.python.org/3.6/whatsnew/3.6.html).

That confused me for a moment: why the version requirement bump to 3.7 here?

It's been a while since I opened this, but I think the rationale was that [Python 3.6 is no-longer-developed](https://peps.python.org/pep-0494/#schedule-last-security-only-release), that it wouldn't make sense to increase a _minimum_ version number to support that.

But I think I've changed my mind.  Some systems could still have valid reasons to use Py3.6, and as long as this change is compatible with those, then that seems like a better lower-bound.  I'll push another commit for that soon.

(they should probably upgrade, but that's up to them, really)